pkg/config/env.go

```diff
@@ -1,21 +1,26 @@
 package config
 
 import (
+	"errors"
 	"os"
 )
 
-var (
-	clientID     string
-	clientSecret string
-)
-
-//LoadEnv set values from .env file
-func LoadEnv() {
```

Comment:
`LoadEnv` is an exported function, but this is not invoked by external packages. So modify to `loadEnv` (unexport)